knack/invocation.py

```diff
@@ -17,6 +21,9 @@
 from .help import CLIHelp
 
 
+logger = get_logger(__name__)
```

Comment:
Don't think you use this?

Reply:
Nope. It did use it, which was the source of all the horribleness. So happy to get rid of it.